StockPriceScraper/Program.cs

```diff
@@ -66,7 +66,7 @@ static bool ReadConfigurationFile()
             {
                 // setup stream to read file
                 var xmldoc = new XmlDocument();
-                xmldoc.Load(@"stock-config.xml");
+                xmldoc.Load("stock-config.xml");
```

Comment:
Didn't know what the '@' does, this was a copy from the tutorial.

Reply:
Yea, "@" can do some other magic